docs/2.develop/relayers/multichain-server.md

```diff
@@ -0,0 +1,71 @@
+:::
+
+## Technical system design
+
+Below is a design diagram of the entire multichain relayer system:
+
+![multichain_relayer_technical_design.png](/docs/multichain_relayer_technical_design.png)
+
+- The [gas station contract](https://github.com/near/multichain-gas-station-contract) and the [MPC signing service contract](https://github.com/near/mpc-recovery/tree/main/contract) are in the green box which take place on NEAR.
+- This multichain relayer server focuses on the purple/blue Multichain Relayer Core Backend Services Box in the middle and the connections to the XChain systems in the red box via RPCs.
+- The XChain Settlement that's happening in the yellow box is currently manual and will be automated in the future.
+
+
+## System workflow
+
+1. The wallet sends a NEAR transaction to the gas station contract that contains 2 actions:
+   1. A transfer of `NEAR` (or FT Transfer in the future) to cover gas on the foreign chain
+   2. A `create_transaction` function call to the gas station contract `canhazgas.testnet` containing the unsigned foreign chain transaction to be signed by the MPC signing service, assuming the unsigned transaction passes validation.
+2. The Gas Station Contract calls the MPC signing service to sign both a funding transaction, which ensures the user's foreign chain account has sufficient gas to execute the desired transaction, and signs the unsigned foreign chain transaction.
+3. Upon receipt of both the signed transactions, the Gas Station Contract emits an event which is picked up by the indexer, which then calls the `/send_funding_and_user_signed_txns` with the 2 signed transactions from the indexer.
+4. The multichain relayer server sends the funding transaction to the foreign chain RPC to fund the user's account with gas.
+5. After the gas funding transaction is confirmed, the multichain relayer server calls the foreign chain RPC again to send the signed transaction initiated by the user in step 1.
+6. The Cross Chain Settlement takes care of selling the extra `NEAR` being sent to the gas station contract for gas tokens on foreign chains as well as bridging the tokens to the other chains. This process is currently manual, but will be automated in partnership with market makers in the future.
+
+## Relayer Server Endpoints
+
+1. `/send_funding_and_user_signed_txns` which handles both
+   1. Funding the user's XChain account with gas from the paymaster treasury account, which is provided as a raw signed transaction
+   2. Sending the user's raw signed transaction (in hexadecimal in EVM case) after the funding transaction has been confirmed on the foreign chain
+2. (MVP only) `/get_balance_for_account` endpoint that gets the native gas token (BNB) balance for an account.
```

Comment:
Remove this from the actual docs since it will be removed in a few weeks

Reply:
`get_balance_for_account` removed